insights/specs.py

```diff
@@ -194,6 +194,7 @@ def docker_container_ids(broker):
 ip_route_show_table_all = sf.simple_command("/sbin/ip route show table all", name="ip_route_show_table_all")
 ipaupgrade_log = sf.simple_file("/var/log/ipaupgrade.log", name="ipaupgrade_log")
 ipcs_s = sf.simple_command("/usr/bin/ipcs -s", name="ipcs_s")
+ip_s_link = sf.simple_command("/sbin/ip -s link", name="ip-s_link")
```

Comment:
`name` should be "ip_s_link", `alias=ip-s_link` should be added to keep compatiable with the old style specs.

Reply:
And, better move the `ip_s_link` ahead of `ipaupgrade_log` to keep the order.